frontend/src/components/common/form/Select.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
```

Comment:
Does this support dynamic list?

Reply:
The select supports passing it an `options` array so yes that's a dynamic list. See the usage in `MapFilterBar.tsx`

```tsx
<Select
    field="classificationId"
    placeholder="View by Classification"
    options={classifications}
```

where the `classifications` array comes from a previous call to the API.
Hope that answers your question...